Update 1.0.0

SaidBySolo opened the pull request:
close #9 
close #10 
close #11 
close #12 
close #16 
close #18 
close #19 
close #34 
close #35 
close #36

SaidBySolo committed:
feat: get meal info

SaidBySolo committed:
feat: choice school, choice date

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

feat: get meal info

SaidBySolo committed:
fix: check result is not found

SaidBySolo committed:
Merge pull request #2 from SaidBySolo/feature

fix: check result is not found

SaidBySolo committed:
chore: add branch

SaidBySolo committed:
Merge branch 'develop' of https://github.com/SaidBySolo/schoolbot into develop

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
style: edit typo

SaidBySolo committed:
chore: not using env

SaidBySolo committed:
fix: apply request changes

SaidBySolo committed:
chore: add env

SaidBySolo committed:
Merge pull request #13 from SaidBySolo/develop

feat: meal info

SaidBySolo committed:
chore: add build config

SaidBySolo committed:
refactor: meal little optimization,  feat: academic_calendar get schoolinfo

SaidBySolo committed:
feat: academic_calendar complete

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

feat: academic_calendar

SaidBySolo committed:
style: rename filename

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
style: rename func name

SaidBySolo committed:
style: rename filename

SaidBySolo committed:
fix: check channel

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
Merge pull request #15 from SaidBySolo/develop

refactor: if statement, feat: academic_schedule

kndlr committed:
feat: add iamschool

kndlr committed:
Revert "chore: add build config"

This reverts commit e00e7f1edc978b04bda198dd0fa4d22b321126df.

SaidBySolo committed:
Merge pull request #20 from callistoteam/staging

deploy beta

kndlr committed:
feat: iamschool article

kndlr committed:
code refactor

kndlr committed:
pull upstream

kndlr committed:
style: apply code style

kndlr committed:
style(iamschool): apply code style

kndlr committed:
fix: apply request changes

SaidBySolo committed:
feat: install private package

kndlr committed:
Merge pull request #1 from callistoteam/SaidBySolo-patch1

feat: install private package

kndlr committed:
Merge branch 'SaidBySolo-patch1' of https://github.com/callistoteam/schoolbot into develop

kndlr committed:
style: apply code style

kndlr committed:
Merge branch 'develop' of https://github.com/COIN-KR/schoolbot into develop

SaidBySolo committed:
Merge pull request #21 from COIN-KR/develop

feat: iamschool article

SaidBySolo committed:
chore: add run.sh

SaidBySolo committed:
Merge pull request #23 from callistoteam/SaidBySolo-patch1

chore: add run.sh

SaidBySolo committed:
Merge pull request #22 from callistoteam/staging

deploy beta

SaidBySolo committed:
chore: edit build config

SaidBySolo committed:
fix: dir

SaidBySolo committed:
fix: permisson denied

SaidBySolo committed:
fix: permisson denied

SaidBySolo committed:
fix: run.sh

SaidBySolo committed:
chore: shell script not need in this branch

SaidBySolo committed:
Merge branch 'staging' into SaidBySolo-patch1

SaidBySolo committed:
Merge pull request #24 from callistoteam/SaidBySolo-patch1

chore: shell script not need in this branch

SaidBySolo committed:
Merge branch 'beta' into staging

SaidBySolo committed:
chore: this branch not need build config

SaidBySolo committed:
feat: timetable

SaidBySolo committed:
Merge branch 'staging' into develop

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
chore: install private dependency

SaidBySolo committed:
Merge pull request #26 from SaidBySolo/develop

feat: timetable

SaidBySolo committed:
feat: help, fix: if info 200 raise unexpected error

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

feat: help, fix: if info 200 raise unexpected error

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
chore: edit install private package

SaidBySolo committed:
chore: edit typo

SaidBySolo committed:
chore: edit typo

SaidBySolo committed:
chore: edit build setup

SaidBySolo committed:
Merge branch 'master' into staging

SaidBySolo committed:
Merge branch 'staging' into develop

SaidBySolo committed:
style: edit typo

SaidBySolo committed:
Merge pull request #27 from SaidBySolo/develop

feat: help, fix: if info 200 raise unexpected error

SaidBySolo committed:
feat: add general cogs

SaidBySolo committed:
style: change version name

SaidBySolo committed:
style: change presence sentence

SaidBySolo committed:
Merge pull request #30 from callistoteam/SaidBySolo-patch-1

Ready to release 1.0.0

SaidBySolo committed:
style: edit typo

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
feat: add timetable command

SaidBySolo committed:
fix: apikey error

kndlr committed:
feat: setting, search, db

kijk2869 commented on schoolbot/cogs/general/help.py:
```diff
@@ -0,0 +1,33 @@
+            name="게시물",
+            value="설명:해당학교의 아이엠스쿨 게시물을 가져옵니다.\n인자값: 학교명(필수)\n예시:\n?게시물 인천기계공업고등학교",
+            inline=False,
+        )
```
```suggestion
        embed = discord.Embed(title="도움말", description="접두사: ``?``") \
            add_field(
                name="급식",
                value="설명:해당학교의 급식정보를 알려줍니다. 날짜가 주어지지 않았을경우 현재 날짜로 가져옵니다.\n인자값: 학교명(필수) 날짜(선택)\n예시:\n?급식 인천기계공업고등학교\n?급식 인천기계공업고등학교 20200810",
                inline=False,
            ) \
            .add_field(
                name="시간표",
                value="설명:해당학교의 시간표를 알려줍니다. 날짜가 주어지지 않았을경우 현재 날짜로 가져옵니다. 현재 고등학교는 가져올수 없습니다.\n인자값: 학교명(필수) 학년(필수) 반(필수) 날짜(선택)\n예시:\n?시간표 구월중학교 2 1\n?시간표 구월중학교 2 1 20200810",
                inline=False,
            ) \ 
            .add_field(
                name="학사일정",
                value="설명:해당학교의 학사정보를 알려줍니다. 날짜가 주어지지 않았을경우 현재 날짜로 가져옵니다.\n인자값: 학교명(필수) 날짜(선택)\n예시:\n?학사일정 인천기계공업고등학교\n?학사일정 인천기계공업고등학교 20200808",
                inline=False,
            ) \ 
            .add_field(
                name="게시물",
                value="설명:해당학교의 아이엠스쿨 게시물을 가져옵니다.\n인자값: 학교명(필수)\n예시:\n?게시물 인천기계공업고등학교",
                inline=False,
            )
```

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                    # traceback.format_exc()
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                # traceback.format_exc()
+                return await msg.edit(
+                    embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                )
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                    # traceback.format_exc()
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                # traceback.format_exc()
+                return await msg.edit(
+                    embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                )
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                    # traceback.format_exc()
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                return await msg.edit(
+                    embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                )
+
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+                    # traceback.format_exc()
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
```
Can cause code duplicate

Use `discord.on_error` or `await Client.on_error`

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+        elif "고등학교" in SN:
+            return await msg.edit(
+                embed=discord.Embed(title="죄송합니다. 현재 고등학교는 지원하지않습니다.")  # 고등학교 지원할때 빼면됨
+            )
```
`scclass` can be undefined

```suggestion
        if "초등학교" in SN:
            scclass = "els"
        elif "중학교" in SN:
            scclass = "mis"
        elif "고등학교" in SN:
            return await msg.edit(
                embed=discord.Embed(title="죄송합니다. 현재 고등학교는 지원하지않습니다.")  # 고등학교 지원할때 빼면됨
            )
        else:
            return await msg.edit(
                embed=discord.Embed(title="죄송합니다. 지원하지 않는 학교입니다.")
            )
```

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                if isinstance(e, DataNotFound):
+                    return await msg.edit(
+                        embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                    )
```
```suggestion
            except DataNotFound:
                return await msg.edit(
                    embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
                )
            except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                    else:
+                        choice = scinfo.data[num]
+                        AE = choice["ATPT_OFCDC_SC_CODE"]
+                        SE = choice["SD_SCHUL_CODE"]
```
```suggestion
                else:
                    fetch_msg = await ctx.fetch_message(response.id)
                    try:
                        num = int(fetch_msg.content) - 1
                    except ValueError:
                        return await msg.edit(
                            embed=discord.Embed(title="잘못된값을 주셨습니다. 처음부터 다시 시도해주세요.")
                        )
                    
                    choice = scinfo.data[num]
                    AE = choice["ATPT_OFCDC_SC_CODE"]
                    SE = choice["SD_SCHUL_CODE"]
```

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                    except ValueError:
+                        return await msg.edit(
+                            embed=discord.Embed(title="잘못된값을 주셨습니다. 처음부터 다시 시도해주세요.")
+                        )
```
Hardcoded

Use `str.isdigit()`

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+            if isinstance(e, DataNotFound):
+                return await msg.edit(
+                    embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                )
```
```suggestion
        except DataNotFound:
            return await msg.edit(
                embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
            )
        except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                if isinstance(e, HTTPException):
+                    return await msg.edit(
+                        embed=discord.Embed(title="HTTP 요청 오류입니다", description=f"{e}")
+                    )
```
```suggestion
            except HTTPException:
                return await msg.edit(
                    embed=discord.Embed(title="HTTP 요청 오류입니다", description=f"{e}")
                )
            except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
+            else:
+                if scinfo == []:
```
```suggestion
                if not scinfo:
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                        )
+                    )
+                else:
+                    if len(scinfo) > 1:
```
```suggestion
                    if scinfo:
```

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
+            if len(scinfo.data) > 1:
```
```suggestion
            if scinfo.data:
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                                    embed=discord.Embed(
+                                        title="잘못된값을 주셨습니다. 처음부터 다시 시도해주세요."
+                                    )
+                                )
```
Hardcoded

Use `str.isdigit()`

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                                    )
+                                )
+                            else:
+                                choice = scinfo[num]
```
```suggestion
                            choice = scinfo[num]
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                return await msg.edit(
+                    embed=discord.Embed(title="HTTP 요청 오류입니다", description=f"{e}")
+                )
+            else:
```
```suggestion
        except HTTPException:
            return await msg.edit(
                embed=discord.Embed(title="HTTP 요청 오류입니다", description=f"{e}")
            )
        except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,177 @@
+                    embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                )
+        else:
+            if scarticles == []:
```
```suggestion
            if not scarticles:
```

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                    return await msg.edit(
+                        embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                    )
+                else:
```
```suggestion
            except DataNotFound:
                if isinstance(e, DataNotFound):
                    return await msg.edit(
                        embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
                    )
            except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
+            if len(scinfo.data) > 1:
```
```suggestion
            if scinfo.data:
```

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                        embed=discord.Embed(title="시간 초과입니다. 처음부터 다시 시도해주세요.")
+                    )
+                else:
+                    fetch_msg = await ctx.fetch_message(response.id)
```
Why fetch again?

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,107 @@
+                        embed=discord.Embed(title="시간 초과입니다. 처음부터 다시 시도해주세요.")
+                    )
+                else:
+                    fetch_msg = await ctx.fetch_message(response.id)
```
Why fetch again?

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                    except ValueError:
+                        return await msg.edit(
+                            embed=discord.Embed(title="잘못된값을 주셨습니다. 처음부터 다시 시도해주세요.")
+                        )
```
Hardcoded

Use `str.isdigit`

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                    else:
+                        choice = scinfo.data[num]
+                        AE = choice["ATPT_OFCDC_SC_CODE"]
+                        SE = choice["SD_SCHUL_CODE"]
```
```suggestion
                    choice = scinfo.data[num]
                    AE = choice["ATPT_OFCDC_SC_CODE"]
                    SE = choice["SD_SCHUL_CODE"]
```

wonderlandpark reviewed (changes requested):
embed에서 역할 색이든 예쁜 색이든 색을 좀 넣으면 좋을거 같습니다.
또한 저번에 말씀드렸던 footer에 timestamp를 넣어주세요

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+                return await msg.edit(
+                    embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                )
+            else:
```
```suggestion
        except DataNotFound:
            return await msg.edit(
                embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
            )
        except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,108 @@
+        await msg.edit(
+            embed=discord.Embed(
+                title=f"{scmeal.SCHUL_NM}의 급식입니다.\n\n{meal_day[0:4]}년 {meal_day[4:6]}월 {meal_day[6:8]}일",
+                description=meal,
```
```suggestion
        await msg.edit(
            embed=discord.Embed(
                title=f"{scmeal.SCHUL_NM}의 급식입니다.\n\n{meal_day[0:4]}년 {meal_day[4:6]}월 {meal_day[6:8]}일",
                description=scmeal.DDISH_NM.replace("<br/>", "\n"),
```

SaidBySolo commented:
> embed에서 역할 색이든 예쁜 색이든 색을 좀 넣으면 좋을거 같습니다.
> 또한 저번에 말씀드렸던 footer에 timestamp를 넣어주세요

#35 이슈에있습니다. 참고해주세요

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+                    return await msg.edit(
+                        embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                    )
+                else:
```
```suggestion
            except DataNotFound:
                return await msg.edit(
                    embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
                )
            except Exception as e:
```

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+                    return await msg.edit(
+                        embed=discord.Embed(title="알수없는 오류입니다", description=f"{e}")
+                    )
+            if len(scinfo.data) > 1:
```
```suggestion
            if scinfo.data:
```

SaidBySolo committed:
feat: change code owner

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+                        embed=discord.Embed(title="시간 초과입니다. 처음부터 다시 시도해주세요.")
+                    )
+                else:
+                    fetch_msg = await ctx.fetch_message(response.id)
```
Why fetch again?

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+                    except ValueError:
+                        return await msg.edit(
+                            embed=discord.Embed(title="잘못된값을 주셨습니다. 처음부터 다시 시도해주세요.")
+                        )
```
Hardcoded

Use `str.isdigit()`

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+                        choice = scinfo.data[num]
+                        AE = choice["ATPT_OFCDC_SC_CODE"]
+                        SE = choice["SD_SCHUL_CODE"]
+                        SN = choice["SCHUL_NM"]
```
```suggestion
                    choice = scinfo.data[num]
                    AE = choice["ATPT_OFCDC_SC_CODE"]
                    SE = choice["SD_SCHUL_CODE"]
                    SN = choice["SCHUL_NM"]
```

wonderlandpark commented:
> > embed에서 역할 색이든 예쁜 색이든 색을 좀 넣으면 좋을거 같습니다.
> > 또한 저번에 말씀드렸던 footer에 timestamp를 넣어주세요
> 
> #35 이슈에있습니다. 참고해주세요

#35 먼저 해결하고 1.0.0으로 올려주시기 바랍니다.

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,139 @@
+            if isinstance(e, DataNotFound):
+                return await msg.edit(
+                    embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
+                )
```
```suggestion
        except DataNotFound:
            return await msg.edit(
                embed=discord.Embed(title="정보가 없습니다. 확인하신후 다시 요청하세요")
            )
        except Exception as e:
```

kijk2869 reviewed (changes requested):
이 코드 리뷰는 깃허브에서 작성되었습니다.
모든 코드 제안에는 주변 코드의 탭 검토가 필요합니다.
코드 제안을 바로 커밋하지 마십시오.

SaidBySolo commented:
> > > embed에서 역할 색이든 예쁜 색이든 색을 좀 넣으면 좋을거 같습니다.
> > > 또한 저번에 말씀드렸던 footer에 timestamp를 넣어주세요
> > 
> > 
> > #35 이슈에있습니다. 참고해주세요
> 
> #35 먼저 해결하고 1.0.0으로 올려주시기 바랍니다.

wait라벨 참고해 주시기 바랍니다. 아직 머지해서는 안되는 풀리퀘입니다.

wonderlandpark commented:
> > > > embed에서 역할 색이든 예쁜 색이든 색을 좀 넣으면 좋을거 같습니다.
> > > > 또한 저번에 말씀드렸던 footer에 timestamp를 넣어주세요
> > > 
> > > 
> > > #35 이슈에있습니다. 참고해주세요
> > 
> > 
> > #35 먼저 해결하고 1.0.0으로 올려주시기 바랍니다.
> 
> wait라벨 참고해 주시기 바랍니다. 아직 머지해서는 안되는 풀리퀘입니다.

확인했습니다

SaidBySolo committed:
Merge remote-tracking branch 'upstream/staging' into develop

SaidBySolo committed:
style: apply request change

SaidBySolo committed:
refactor: apply request changes

kndlr committed:
fix: search strftime

SaidBySolo committed:
refactor: apply request changes

SaidBySolo committed:
refactor: apply request changes

SaidBySolo committed:
style: delete remark

SaidBySolo committed:
refactor: check several schools

SaidBySolo committed:
refactor: apply chnage requests

fix: timezone issue
feat: debug module

SaidBySolo committed:
Merge pull request #38 from SaidBySolo/develop

refactor: apply request changes

SaidBySolo committed:
chore: add dependency

chore: some change build config

SaidBySolo committed:
chore: add env

SaidBySolo committed:
Merge pull request #39 from SaidBySolo/develop

chore: add dependency

SaidBySolo committed:
chore: add branch

SaidBySolo committed:
Merge pull request #40 from SaidBySolo/develop

chore: add branch

SaidBySolo committed:
style: change prefix, delete not using code

SaidBySolo committed:
fix: str - int error

SaidBySolo committed:
feat: check owner, add limit traceback

SaidBySolo committed:
Merge pull request #42 from SaidBySolo/develop

fix: str - int error, feat: handle error, limit traceback log, style: edit help

kndlr committed:
feat: db meal, schedule

SaidBySolo committed:
style: embed styling

SaidBySolo committed:
Merge pull request #44 from SaidBySolo/develop

style: embed styling

SaidBySolo committed:
fix: cant send embed

SaidBySolo committed:
Merge pull request #46 from SaidBySolo/develop

fix: cant send embed

kndlr committed:
feat: db timetable

kijk2869 committed:
style: dependent version management

kijk2869 committed:
style: apply dependabot

kijk2869 committed:
Merge pull request #48 from callistoteam/kijk2869-patch-1

style: dependent version management

kijk2869 committed:
docs: add contact

kndlr committed:
style: apply code style

kndlr committed:
fix: help command

kndlr committed:
fix: apply request changes

wonderlandpark committed:
feat: #36

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

wonderlandpark committed:
fix: request querystring

wonderlandpark committed:
Merge branch 'staging' into i'm-so-sexy

wonderlandpark committed:
fix: hard coding

Co-authored-by: 매리 <mareowst@gmail.com>

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

wonderlandpark committed:
fix: hard coding

Co-authored-by: Ryu JuHeon <saidbysolo@gmail.com>

wonderlandpark committed:
fix: encode uri

wonderlandpark committed:
fix: hard coding

wonderlandpark committed:
feat: os env with schoolbot api token

wonderlandpark committed:
Merge branch 'feature-meal-prerendering' into i'm-so-sexy

wonderlandpark committed:
Merge pull request #54 from callistoteam/i'm-so-sexy

I'm so sexy

wonderlandpark committed:
feat: urllib parse quote

wonderlandpark committed:
Merge branch 'feature-meal-prerendering' of https://github.com/callistoteam/schoolbot into feature-meal-prerendering

wonderlandpark committed:
fix: env name

wonderlandpark committed:
feat: format as black

wonderlandpark committed:
Merge pull request #53 from callistoteam/feature-meal-prerendering

feat: added meal prerendering ( #36 )

wonderlandpark committed:
Merge branch 'staging' into develop

kijk2869 commented:
will reopen

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

kndlr committed:
fix: apply request changes

SaidBySolo committed:
Merge pull request #49 from COIN-KR/develop

feat: aiomysql

wonderlandpark committed:
Update python-app-lint.yml

wonderlandpark committed:
Update python-app-test.yml

Yoruroong committed:
Update README.md

wonderlandpark commented on README.md:
```diff
@@ -1,7 +1,11 @@
 # schoolbot
 
-> A discord bot that informs the school's academic calendar, timetable, and meals in Korea
+> A discord bot that informs the school's academic calendar, timetable, and meals in South Korea
```
Republic of Korea

wonderlandpark commented on schoolbot/cogs/general/help.py:
```diff
@@ -0,0 +1,43 @@
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="도움말", aliases=["help"])
```
aliases: 도움, commands, 명령어 등등

wonderlandpark commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,151 @@
+    async def _meal(self, ctx, school_name: str = None, date: int = None):
+        msg = await ctx.send(
+            embed=discord.Embed(
+                title="정보를 요청합니다 잠시만 기다려주세요.", colour=discord.Colour.blurple()
```
check command syntax first

(`정보를 요청합니다` 잠시만 기다려주세요가 `학교명을 입력해주세요` 보다 먼저 표시됨)

SaidBySolo replied:
의도한것이지만 수정하도록하겠습니다.

wonderlandpark commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,151 @@
+                    AE = choice["ATPT_OFCDC_SC_CODE"]
+                    SE = choice["SD_SCHUL_CODE"]
+            else:
+                # 대충 여따가 쿼리문 적으면 된다는 주석
```
주석 지우시는게 좋을거 같습니다.

wonderlandpark commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,218 @@
+    ):
+        msg = await ctx.send(
+            embed=discord.Embed(
+                title="정보를 요청합니다 잠시만 기다려주세요.", colour=discord.Colour.blurple()
```
check command syntax first

(`정보를 요청합니다 잠시만 기다려주세요`가 `학년과 반정보를 입력해주세요!` 보다 먼저 표시됨)

SaidBySolo replied:
의도한것이지만 수정하겠습니다.

Yoruroong committed:
Update README.md

Yoruroong committed:
fix: apply request changes

fossabot committed:
Add license scan report and status

Signed off by: fossabot <badges@fossa.com>

kijk2869 committed:
Merge pull request #57 from fossabot/add-license-scan-badge

Add license scan report and status

kijk2869 commented on .github/workflows/python-app-lint.yml:
```diff
@@ -23,8 +24,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
-        pip install pytest pytest-asyncio flake8 black
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install flake8 black
```
Installing requirements deleted

kijk2869 commented on schoolbot/cogs/events/error.py:
```diff
@@ -0,0 +1,67 @@
+                f"**Command**: ``{ctx.command}``\n"
+                f"**Bot Permission**: ``{ctx.guild.me.guild_permissions.value}``",
+            )
+            trace_embed.set_footer(text=kst_sft())
```
How about using timestamp

kijk2869 commented on schoolbot/cogs/general/search.py:
```diff
@@ -0,0 +1,109 @@
+                        embed=discord.Embed(title="시간 초과입니다. 처음부터 다시 시도해주세요.")
+                    )
+                else:
+                    fetch_msg = await ctx.fetch_message(response.id)
```
Message refetched

kijk2869 commented on schoolbot/cogs/general/setting.py:
```diff
@@ -0,0 +1,74 @@
+    @commands.command(name="설정")
+    async def _search(self, ctx, key: str = None, *, value: str = None):
+        if key and value:
+            args = " ".join(value.split("|")).split()
```
Same as `args = value.split("|")`

kijk2869 commented on schoolbot/cogs/school/academic_schedule.py:
```diff
@@ -0,0 +1,147 @@
+        )
+        user_data = await db.get_user_data(ctx.author.id)
+        if user_data and (
+            (date == None and school_name == None)
```
```suggestion
            not (date and school_name)
```

kijk2869 commented on schoolbot/cogs/school/article.py:
```diff
@@ -0,0 +1,183 @@
+                await msg.add_reaction("◀")
+                await msg.add_reaction("⏹")
+                await msg.add_reaction("▶")
+                while True:
```
```suggestion
                while not self.bot.is_closed():
```

kijk2869 commented on schoolbot/cogs/school/meal.py:
```diff
@@ -0,0 +1,151 @@
+        )
+        user_data = await db.get_user_data(ctx.author.id)
+        if user_data and (
+            (date == None and school_name == None)
```
```suggestion
            not (date and school_name)
```

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,218 @@
+                            )
+                        )
+                    else:
+                        fetch_msg = await ctx.fetch_message(response.id)
```
Unused variable

kijk2869 commented on schoolbot/cogs/school/timetable.py:
```diff
@@ -0,0 +1,218 @@
+            tt_scname = sctimetable.data[0]["SCHUL_NM"]
+            tt_day = sctimetable.data[0]["ALL_TI_YMD"]
+            await msg.edit(
+                embed=discord.Embed(title=f"{tt_scname}", colour=0x2E3136,)
```
```suggestion
                embed=discord.Embed(title=tt_scname, colour=0x2E3136,)
```

SaidBySolo committed:
fix: question mark is the most used, so it can be inconvenient.

SaidBySolo committed:
style: delete remark

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve converstion

SaidBySolo committed:
style: dupe func name

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
Merge pull request #1 from SaidBySolo/feature

Resolve conversation and fix some problem

SaidBySolo committed:
refactor: unused module

SaidBySolo committed:
style: typo

SaidBySolo committed:
refactor: neispy 3.0 update

SaidBySolo committed:
refactor: neispy update 3.0

SaidBySolo committed:
Merge pull request #2 from SaidBySolo/feature

refactor: neispy update 3.0

SaidBySolo committed:
style: typo

SaidBySolo committed:
style: apply code style

SaidBySolo committed:
chore: check lint

SaidBySolo committed:
refactor: resolve conversation

SaidBySolo committed:
Merge pull request #58 from SaidBySolo/develop

neispy 3.0 update corver, resolve conversation

kijk2869 commented:
이제 슬슬 머지하자 이 pr에만 164커밋이야

Yoruroong commented:
PR에 164커밋있는거 처음봄

SaidBySolo commented:
not resolve https://github.com/callistoteam/schoolbot/pull/28#discussion_r477845290

dependabot[bot] committed:
chore(deps): bump neispy from 2.0.7 to 3.0.0

Bumps [neispy](https://github.com/SaidBySolo/neispy) from 2.0.7 to 3.0.0.
- [Release notes](https://github.com/SaidBySolo/neispy/releases)
- [Commits](https://github.com/SaidBySolo/neispy/commits)

Signed-off-by: dependabot[bot] <support@github.com>

kijk2869 committed:
Merge pull request #60 from callistoteam/dependabot/pip/neispy-3.0.0

chore(deps): bump neispy from 2.0.7 to 3.0.0

kijk2869 committed:
Update README.md

kijk2869 committed:
fix https://github.com/callistoteam/schoolbot/issues/62

kijk2869 committed:
Create python-type-hint.yml

kijk2869 committed:
Update python-type-hint.yml

kijk2869 committed:
Delete python-type-hint.yml

kijk2869 committed:
일해라 라솔

kndlr committed:
docs: default value

wonderlandpark reviewed (approved):
LSTM
